src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -48,9 +48,22 @@ if(ENABLE_TESTS)
     add_subdirectory(test_utils)
 endif()
 
+if(TARGET ov_onnx_frontend)
+    add_subdirectory(frontend/onnx)
+endif()
```

Comment:
I suppose such checks don't work for cases when we build python module based on Developer Package. Dev Package has prefixes like `openvino::` or `IE::` for all the targets

Reply:
fixed